packages/ui/react-ui-editor/src/extensions/markdown/linkPaste.ts

```diff
@@ -15,9 +15,17 @@ const createUrlLink = (url: string): string => {
   return `[${displayUrl}](${url})`;
 };
 
-const formatUrlForDisplay = (url: string): string => {
-  const withoutProtocol = url.replace(/^https?:\/\//, '');
-  return truncateQueryParams(withoutProtocol);
+export const formatUrlForDisplay = (url: string): string => {
+  // Remove protocol (http:// or https://)
+  let formattedUrl = url.replace(/^https?:\/\//, '');
+
+  // Remove 'www.' if at the beginning of the URL
```

Comment:
maybe create an issue upstream and link to it here or at the very least link to the dxos issue this is resolving to explain why this is necessary